flaeg.go

```diff
@@ -14,6 +14,8 @@ import (
 	"text/template"
 	"time"
 
+	parse "github.com/containous/flaeg/parse"
+
```

Comment:
could you remove this empty line.

Reply:
and remove the alias

Reply:
Do you use golint? 
Any specific configuration? 
I get a few warnings regarding comments and error message formats.